src/main/java/org/killbill/billing/client/KillBillHttpClient.java

```diff
@@ -325,7 +325,7 @@ public Response doOptions(final String uri, final Multimap<String, String> optio
     }
 
     private <T> T doPrepareRequestAndMaybeFollowLocation(final String verb, final String uri, final Object body, final Multimap<String, String> optionsRo, final Multimap<String, String> optionsForFollow, final int timeoutSec, final Class<T> clazz, final boolean followLocation) throws KillBillClientException {
-        final Multimap<String, String> options = HashMultimap.<String, String>create(optionsRo);
+        final Multimap<String, String> options = HashMultimap.create(optionsRo);
```

Comment:
I'm talking much older :-)

I don't have the bug report handy but it was affecting early updates (pre u20?) that would come on old CentOS versions (and I know one Kill Bill user still on it!).